src/backend/distributed/utils/ruleutils_95.c

```diff
@@ -7383,4 +7383,38 @@ generate_operator_name(Oid operid, Oid arg1, Oid arg2)
 	return buf.data;
 }
 
+
+/*
+ * generate_qualified_relation_name
+ *		Compute the name to display for a relation specified by OID
+ *
+ * As above, but unconditionally schema-qualify the name.
+ */
+char *
+generate_qualified_relation_name(Oid relid)
```

Comment:
This would probably have to be added to the 9.6 version of ruleutils as well. What if we place it somewhere else to avoid repeating it?